providers/mso/mso_provider.go

```diff
@@ -55,6 +55,9 @@ func (p MSOProvider) GetResourceConnections() map[string]map[string][]string {
 			"schema_template":    []string{"template_name", "name"},
 			"schema_template_bd": []string{"bd_name", "name"},
 		},
+		"schema_template_anp": {
```

Comment:
also uncomment all the previous reference to `schema_template_anp`

Reply:
Done